pyproject.toml

```
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "dj-registry"
+version = "0.1.0"
+description = "A simple, easy access key-value registry for Django."
+authors = ["yychen <ctchen@gmail.com>"]
```

The author's name should be the same as the current [`setup.py`](https://github.com/yychen/dj-registry/blob/master/setup.py).
```suggestion
authors = ["Tom Chiung-ting Chen <ctchen@gmail.com>"]
```

---

should we put all four of us? :D :D

---

it's your call boss!